src/config/configuration.f90

```diff
@@ -7,40 +7,40 @@
 !!
 !!------------------------------------------------
 module config_mod
-    
+
```

Comment:
even after merging #17, I'm still seeing all the trailing whitespace changes in this pull request, might be best to reopen the pull request now that #17 is merged(?)